### PR TITLE
Notes and ideas for migration and template

### PR DESCRIPTION
**Pull request by bdenham**
## Purpose of this pull request

Just using this PR to communicate notes and ideas for things we should do in the `https://github.com/AdobeDocs/dev-site-documentation-template` as well as ways to handle troublesome markdown --> mdx conversions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,30 @@ View the [demo](https://adobedocs.github.io/dev-site-documentation-template/) ru
 
 ## Where to ask for help
 
-The slack channel #adobeio-onsite-onboarding is our main point of contact for help. Feel free to join the channel and ask any questions. 
+The slack channel #adobeio-onsite-onboarding is our main point of contact for help. Feel free to join the channel and ask any questions.
 
-## How to develop
+## How to build locally
+
+This site uses the Yarn package manager to manage dependencies and run scripts. Open a terminal at the project root and enter these commands in the following order:
```
Comment (jeff-matthews) [inline README.md:13]: What are some of the benefits of using yarn over npm?

File: README.md [hunk 2]
```diff
@@ -6,17 +6,30 @@ View the [demo](https://adobedocs.github.io/dev-site-documentation-template/) ru
...
+
+## How to fix MDX build errors
+
+To fix common markdown errors within MDX files, open a terminal at the project root and enter the following command:
 
-For local development, simply use :
 ```
-$ npm install
-$ npm run dev
+yarn fix-markdown src/pages
```
Comment (jeff-matthews) [inline README.md:24]: Does this include markdown linting errors or does it refer to the use of MDX tags within markdown?

File: package.json [hunk 3]
```diff
@@ -22,6 +23,7 @@
     "dev:https": "gatsby develop --https --host localhost.corp.adobe.com --port 9000",
     "build": "gatsby build",
     "serve": "gatsby serve",
+    "fix-markdown": "markdown-cleaner",
```
Comment (jeff-matthews) [inline package.json:26]: Is this the original package you worked on for parliament?

https://github.com/adobe/parliament-markdown-cleaner

Comment (bdenham) [reply package.json:26]: Yeah, but it wasn't in this separate package at the time. See the full explanation in our migration channel. :)

File: README.md (hunk 2 above)
Comment (bdenham) [reply README.md:24]: Just fixes the problematic markdown syntax. Nothing to do with linting, sorry to say.

**Pull request closed without merging**